tests/components/apsystems/conftest.py

```diff
@@ -23,7 +23,7 @@ def mock_setup_entry() -> Generator[AsyncMock]:
 
 
 @pytest.fixture
-def mock_apsystems() -> Generator[AsyncMock, None, None]:
+def mock_apsystems() -> Generator[AsyncMock]:
```

Comment:
This one was incorrect in the original type hint.
I suggest to fix now:
```suggestion
def mock_apsystems() -> Generator[Mock]:
```